lib/ansible/modules/system/reset_ssh_session.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
```

Comment:
I am looking in other modules under lib/ansible/system and they have a shabeng...